src/book/01-team/05-robocup/03-debriefs.mdx

```diff
@@ -6,6 +6,126 @@ description: Debriefs and takeaways from our performances at RoboCup.
+
+#### Vision
+
+- Red camera. Possible hardware issue? Swapping the cable fixed the issue, but this may have reset the automatic white balance
+- New mesh is too computationally expensive for the NUCs that we have. Potentially go down to one camera per robot and lower the number of images received from the camera
+- The bottom of the mesh, near the robot's feet, bends around, possibly causing issues with ball detection near the feet
+- Hoc was slightly off - odometry or camera calibration
+- NUsight was unreliable for testing vision
+- Theta was in check-in, not carry-on, so couldn’t collect data until later. Did get one image earlier using a theta from CIT Brains
+
+#### NUsight
+
+- Difficulty connecting to NUsight
+
+#### Localisation and Odometry
+
+- No localisation
+- Hoc was slightly off for vision
+
+#### Miscellanious
+
+- No robots until end of first competition day due to missing luggage
+
+### What went well
+
+#### Hardware
+
+- Didn’t replace a single servo at the competition
+- Padding was effective for keeping the robot protected and for colour requirement in inspection
+
+#### Motions
+
+- The robots were able to walk stabily!!
+- Getups basically worked flawlessly
+
+#### Networking
+
+- Able to connect to GameController for most of the games
+
+#### Odometry
+
+- Pitch and roll were good, better than the previous year
+
+#### Vision
+
+- Fine tuning for vision worked well despite not having our own Theta camera
+
+#### Teamwork and Moral
+
+- Got robot ready for qualification very quickly
+- We all came to the games - had three different robot handlers for four games
+- Got lots of promotional footage of the robot
+- We made the quarter finals despite not having two whole set up days and the first competition day to test the system and networking
+- Within 30 minutes of getting our robots from the airport, the robots were walking and getting up
+- Good preparation before the competition was vital to the team being able to perform at all when the robots arrived at the end of the first competition day
+- Overall made good progress compared to RoboCup 2019 Sydney
+- On par or better than Team-KMUTT, UTRA, Hamburg Bit-Bots, Bold Hearts, ITAndroids, RoboFEI
+- Students at home were around to help with vision fine-tuning
+
+### What’s Next
+
+#### Networking
+
+- System configuration tool
+- Configurable headless mode, lightweight, if no system config tool due to it being tedious to set up networking
+- Team Robot communication – only one robot connects to GameController and then sends information to the other robots
+- Make GameController connection more robust
+- See how the small robots do team communication
+
+#### Hardware
+
+- Find a replacement for CM740’s quick so we can buy or make them before the end of the year.
+  - Redesign NUsense to only contain the core functionality – then additional functionality can be added with shields – modular system – modules that you can plug in
```

Comment:
I think that this sentence reads a little awkwardly. Also, what are shields in this context? Should I know that?

Reply:
I think i heard Mikyla describe them as daughter boards?